lib/scheduler.pm

```diff
@@ -25,6 +25,7 @@ use File::Basename;
 use testapi qw(get_var set_var);
```

Comment:
I mean, we don't import `diag` here. It should be enough, as in unit tests we don't call `diag` directly.

Reply:
I noticed too, that works, thanks, perfect.